app/src/main/res/layout/screen_app_lock.xml

```diff
@@ -108,7 +108,16 @@
       <TextView
         android:id="@+id/applock_error"
         style="@style/Clinic.LoginPinErrorText"
-        android:text="@string/applock_error_incorrect_pin" />
+        android:text="@string/applock_error_incorrect_pin"
+        tools:visibility="visible" />
+
+      <Button
+        android:id="@+id/applock_forgotpin"
+        style="@style/Widget.AppCompat.Button.Borderless"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/applock_forgotpin"
+        android:textColor="?colorAccent" />
```

Comment:
FYI you can also use `Widget.AppCompat.Button.Borderless.Colored` instead of setting the text color separately